### PR TITLE
Feature/admin 191 anagkazo banking

### PR DESCRIPTION
**Pull request by dabick14**

### CODE REVIEW

**Review by jaedag — CHANGES_REQUESTED** [review]
Let's change the name of the property

File: api/src/schema/services.graphql [hunk 1]
```diff
@@ -197,6 +198,7 @@ extend type GatheringService {
         WHERE record.noServiceReason IS NULL
           AND record.bankingSlip IS NULL
           AND (record.transactionStatus IS NULL OR record.transactionStatus <> 'success')
+          AND record.tellerConfirmationAt IS NULL
```
[the remark this answers is not in the document]
Comment (dabick14) [reply api/src/schema/services.graphql:201]: Ok

File: api/src/schema/services.graphql [hunk 2]
```diff
@@ -212,7 +214,7 @@ extend type GatheringService {
       WITH date() as today, this
       WITH  today.weekDay as theDay, today, this
       WITH date(today) - duration({days: (theDay - 2)}) AS startDate, this
-      WITH [day in range(0, 4) | startDate + duration({days: day})] AS dates, this
+      WITH [day in range(0, 5) | startDate + duration({days: day})] AS dates, this
```
Comment (jaedag) [inline api/src/schema/services.graphql:217]: What's this about?

**Review by jaedag — APPROVED** [review]

**Comment by dabick14** [comment]
> I don't understand why there is a merge commit? 

There was a deploy in between and so I pulled from the deploy branch

**Pull request merged**